src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -116,18 +129,22 @@ export const WorkspaceCreator = (props: WorkspaceCreatorProps) => {
           color="subdued"
           hasShadow={false}
         >
-          {application && savedObjects && (
-            <WorkspaceForm
-              application={application}
-              savedObjects={savedObjects}
-              onSubmit={handleWorkspaceFormSubmit}
-              operationType={WorkspaceOperationType.Create}
-              permissionEnabled={isPermissionEnabled}
-              dataSourceManagement={dataSourceManagement}
-              availableUseCases={availableUseCases}
-              defaultValues={defaultWorkspaceFormValues}
-            />
-          )}
+          {application &&
+            savedObjects &&
+            // Default values only worked for component mount, should wait for isOnlyAllowEssential and availableUseCases loaded
+            isOnlyAllowEssential !== undefined &&
+            availableUseCases !== undefined && (
```

Comment:
Maybe assign this to a readable name?
```suggestion
          {application &&
            savedObjects &&
            // Default values only worked for component mount, should wait for isOnlyAllowEssential and availableUseCases loaded
            isOnlyAllowEssential !== undefined &&
            availableUseCases !== undefined && (
```